projects/packages/sync/src/class-actions.php

```diff
@@ -163,6 +174,22 @@ public static function add_sender_shutdown() {
 		}
 	}
 
+	/**
+	 * Immediately sends actions on init for the current dedicated Sync request.
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function add_dedicated_sync_sender_init() {
+		if ( apply_filters(
```

Comment:
If we have already determined Dedicated Sync is enabled and this is a dedicated sync request, why are we checking if the sender should initialize? It isn't necessary.

Reply:
👍